SCAutolib/models/log.py

```diff
@@ -0,0 +1,40 @@
+from SCAutolib import logger
+from contextlib import contextmanager
+import re
+
+SECURE_LOG = '/var/log/secure'
```

Comment:
This is not used anywhere, I expect it was supposed to be impotred to tests, however, it's hardcoded in tests itself. Therefore, it's redundant and can be removed.